### PR TITLE
Bulk transfers design - continued

### PR DESCRIPTION
**Pull request by elnyry**
- Fulfil overview sequence diagram
- New: Bulk Processing Handler diagrams
- Refactoring naming, titles, links to other SDs
- Fixes to existing designs
- Updated ERD
- Readme File updated to include design changes for including Object store
- Updated Architecture diagram and sequence diagrams to include object store

**Note**: The SVGs present in the PR are just to help with the PR review. Once it is approved, I'll delete them before merging.

### CODE REVIEW

File: mojaloop-technical-overview/central-bulk-transfers/README.md [hunk 1]
```diff
@@ -12,86 +12,239 @@ The Bulk Transfers scenario is described in the API Definition document regardin
   
 ## 1. Introduction
 
-The Bulk Transfers process is discussed in section 6.10 of the API Definiton 1.0 document, depicted in [Figure 60](./Figure60-Example-Bulk-Transfer-Process-Spec1.0.png). The key items implied in the specification in its current version 1.0 are that
+The Bulk Transfers process is discussed in section 6.10 of the API Definition 1.0 document, depicted in [Figure 60](./Figure60-Example-Bulk-Transfer-Process-Spec1.0.png). The key items implied in the specification in its current version 1.0 are that
```
Comment (mdebarros) [inline mojaloop-technical-overview/central-bulk-transfers/README.md]: I do not see this image `./Figure60-Example-Bulk-Transfer-Process-Spec1.0.png` included as part of the PR.

File: mojaloop-technical-overview/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-1.1.0-bulk-prepare-overview.plantuml [hunk 2]
```diff
@@ -146,7 +152,7 @@ group DFSP1 sends a Bulk Prepare Transfer request to DFSP2
             type: application/json
             content: {
                 headers: <bulkTransferHeaders>,
-                payload: <bulkTransferMessage>
+                payload: <bulkTransferMessage_Reference>
```
[the remark this answers is not in the document]
Comment (elnyry) [reply mojaloop-technical-overview/central-bulk-transfers/assets/diagrams/sequence/seq-bulk-1.1.0-bulk-prepare-overview.plantuml]: Will make this change

File: discussions/discussions.md [hunk 3]
```diff
@@ -0,0 +1 @@
+New folder for discussions.
```
[the remark this answers is not in the document]
Comment (elnyry) [reply discussions/discussions.md]: Change made in the new PR for bulk

**Comment by elnyry** [comment]
Closing this PR as this has some outdated changes. New PR to follow for bulk changes is #68

**Pull request closed without merging**